scipy/linalg/tests/test_lapack.py

```diff
@@ -662,8 +662,8 @@ def test_lartg():
 
         if np.iscomplexobj(g):
             assert_allclose(sn, -4.0j/5.0)
-            assert_(type(r) == complex)
-            assert_(type(cs) == float)
+            assert_(isinstance(r, complex))
+            assert_(isinstance(cs, float))
```

Comment:
this change would probably be better in a separate PR - it does look like a Flake8 improvement, but not a typo.

Reply:
This is a separate commit, attempting to fix linter errors in CI jobs.